easybuild/easyconfigs/b/BEEF/BEEF-0.1.1-iccifort-2020.4.304.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'BEEF'
+version = '0.1.1'
+
+homepage = 'svn://suncatls1.slac.stanford.edu/beef'
```

Comment:
what about using https://confluence.slac.stanford.edu/display/SUNCAT/BEEF+Functional+Software ?

Reply:
yeah I'm not sure how i overlooked that svn url in there.